Client/Frontend/Settings/HomepageSettings/WallpaperSettings/v2/WallpaperSettingsHeaderView.swift

```diff
@@ -0,0 +1,161 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct WallpaperSettingsHeaderViewModel {
+    var title: String
+    var titleA11yIdentifier: String
+
+    var description: String?
+    var descriptionA11yIdentifier: String?
+
+    var buttonTitle: String?
+    var buttonA11yIdentifier: String?
+    var buttonAction: (() -> Void)?
+}
+
+class WallpaperSettingsHeaderView: UICollectionReusableView, ReusableCell {
+
+    private struct UX {
+        static let stackViewSpacing: CGFloat = 4.0
+        static let topBottomSpacing: CGFloat = 16.0
+    }
+
+    private var viewModel: WallpaperSettingsHeaderViewModel?
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
+
+    // Views
+    fileprivate lazy var contentStackView: UIStackView = .build { stackView in
```

Comment:
I believe we have `private_over_fileprivate` Swiftlint rule enabled so you should be getting a warning about this. Is there a reason we want these to be `fileprivate`?